### ISSUE TITLE
[shellformat.path] not configured

### ISSUE THREAD

**Issue by xenadmin** [foxundermoon/vs-shell-format#23]
Hi, I'm trying out your Visual Studio Code Extension, but I can't get it to run. I guess I must be stupid. 
I'm on Windows 10 1803 x64. Visual Studio Code: 
```
Version: 1.29.1 (user setup)
Commit: bc24f98b5f70467bc689abf41cc5550ca637088e
Datum: 2018-11-15T19:13:36.375Z
Electron: 2.0.12
Chrome: 61.0.3163.100
Node.js: 8.9.3
V8: 6.1.534.41
Architektur: x64
```
I've downloaded "https://github.com/mvdan/sh/releases/download/v2.6.2/shfmt_v2.6.2_windows_amd64.exe" and placed it in a folder. 
I then added the path to my custom user settings:
![grafik](https://user-images.githubusercontent.com/12057348/49738004-95962200-fc8e-11e8-9210-39a841494668.png)
But regardless what path I enter, I get the error message: `[shellformat.path] not configured`
Things I tried:
```
"C:\#Apps\shfmt_v2.6.2_windows_amd64.exe"
"C:\\#Apps\\shfmt_v2.6.2_windows_amd64.exe"
"C:/#Apps/shfmt_v2.6.2_windows_amd64.exe"
"C:\shfmt_v2.6.2_windows_amd64.exe"
"C:\\shfmt_v2.6.2_windows_amd64.exe"
"C:/shfmt_v2.6.2_windows_amd64.exe"
"C:\shfmt.exe"
"C:\\shfmt.exe"
"C:/shfmt.exe"
```
I've no idea what basic thing I'm doing wrong. Can you help me please?

**Comment by simonarons** [comment]
I have this issue as well, and I have tried _exactly_ those examples above, except I used `C:\bin` instead of `C:\#Apps`. And I got the exact same result.

It worked a while ago, though. But I can’t remember if it stopped working with a VSCode update or the extension update.

**Comment by foxundermoon** [comment]
i was tested  ,it's ok .
```
{
    "shellformat.path": "C:/Users/fox/Downloads/shfmt_v2.6.2_windows_amd64.exe"
}
```

**Comment by foxundermoon** [comment]
and it's  also can work ,  but seem always  path error tips.

```
{
    "shellformat.path": "C:\\Users\\fox\\Downloads\\shfmt_v2.6.2_windows_amd64.exe"
}
```

**Comment by xenadmin** [comment]
I did try that, same error: `[shellformat.path] not configured please download  https://github.com/mvdan/sh/releases or go get -u mvdan.cc/sh/cmd/shfmt to install`
![grafik](https://user-images.githubusercontent.com/12057348/49804278-6fd25100-fd52-11e8-8090-9727dcca2a2c.png)

**Comment by xenadmin** [comment]
> and it's also can work , but seem always path error tips.
You mean it works, and the error message is wrong? How can I prove the extension is working as it should?

**Comment by foxundermoon** [comment]
fixed on v2.0.2
